src/twisted/web/http.py

```diff
@@ -1890,12 +1889,14 @@ class _ChunkedTransferDecoder:
         state transition this is truncated at the front so that index 0 is
         where the next state shall begin.
 
-    @ivar _start: While in the C{'CHUNK_LENGTH'} state, tracks the index into
-        the buffer at which search for CRLF should resume. Resuming the search
-        at this position avoids doing quadratic work if the chunk length line
-        arrives over many calls to C{dataReceived}.
+    @ivar _start: While in the C{'CHUNK_LENGTH'} and C{'TRAILER'} states,
+        tracks the index into the buffer at which search for CRLF should resume.
+        Resuming the search at this position avoids doing quadratic work if the
+        chunk length line arrives over many calls to C{dataReceived}. Not used
+        in any other state.
 
-        Not used in any other state.
+    @ivar _trailerHeaders: Accumulates trailer headers. Not passed outside the
+        decoder class for a while.
```

Comment:
Maybe we can just document what it does.

This is private, so people should not use it.

I have created a follow ticket to look into exposing them.
and added a link.

So if anyone that wants to use this member, should look into defining the public API

```suggestion
    @ivar _trailerHeaders: Accumulates raw/unparsed trailer headers.
        See https://github.com/twisted/twisted/issues/12014
```